README.md

```diff
@@ -1,13 +1,13 @@
-# Project Name
+# Guess Who game project
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+This weeks project was to create a Guess Who game by using Javascript, and foremost arrays and objects.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+I had trouble understanding how to start the project at first. The steps in the brief were helpful, but I still felt confused. It helped a bit when I saw how my team-mates hade done it in their projects. I coded along with the lesson on wednesday, and that helped a lot! But later on I started to get confused again, mainly because I swithed focus between the pre-written code, the lessons and the steps. Once again I got great help from my team-mates to solve it. The rest I managed to figure out by looking up info on Stack Overflow and other sites, and one time I looked up someone else project on GitHub to see how they solved the problem. It turned out that I was mostly on the correct path, and that most of my errors were typos or that I forgot to put an alert() in the correct place and such. If I had more time I would love to add an counter and a timer, and to show the result on the win-or-lose-page. I would also like to style it a bit more, and it would be fun to add my own family instead. Also sometimes the code doesn't work the way we were shown, and I would like to have time to figure out why. But the code is working now as it is, and I'm satisfied with that.
```

Comment:
Smart that you use several ways to get help, ask your team, checking StackOverflow and looking at other's solutions. Remember that the lectures mostly only show one approach. There's always several ways of doing something, and the most important thing is that you understand the approach you have chosen.